frontend/src/components/contentDisplay/EmbedDisplay.vue

```diff
@@ -0,0 +1,43 @@
+<!-- Displays an embed content in the editor. -->
+<template>
+  <div class="w-full h-full cursor-pointer" @click="emitEdit()">
```

Comment:
could this prove difficult, since you'd need to click on the iframe and e.g. interact with it (start a video, or something similar) in order to edit it?

Can we prevent interaction with the iframe in edit mode on first click (so can only interact if it's selected), or display an "invisible" overlay (or just partially-transparent) to avoid interacting with the iframe (on first click)?

Reply:
Is tested and works as intended. In editMode the iframe in this div has pointer-events-none which prevents the clicking of the iframe.